lib/tasks/add_navigation_links.rake

```diff
@@ -25,78 +25,91 @@ namespace :navigation_links do
       icon: reading_icon,
       display_only_when_signed_in: true,
       position: 0,
+      section: 0,
```

Comment:
I'm not saying you have to do this but `enum`s allow us to also assign the values as symbols, which would make this file a bit more readable to the casual reader.

```ruby
nl = NavigationLink.first
nl.section
#=> "default"
nl.section = :other
nl.save
nl.reload.section
#=> "other"
nl.section_before_type_cast
#=> 1
```